templates/deploy-openstack.tpl

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/sh
+set -euxo pipefail
+
+declare -A virtual_environments=(
+  ["kayobe"]="$HOME/venvs/kayobe/bin/activate"
+  ["openstack"]="$HOME/src/openstack-config/ansible/openstack-config-venv/bin/activate"
+)
+
+declare -A config_directories=(
+  ["kayobe"]="$HOME/src/kayobe-config"
+  ["openstack"]="$HOME/src/openstack-config"
+)
+
+function activate_virt_env () {
+  set +u
+  source $${virtual_environments[$1]}
+  set -u
+}
+
+function activate_kayobe_env () {
+  set +u
+  source $${config_directories[kayobe]}/kayobe-env --environment ci-multinode
+  set -u
+}
+
+activate_virt_env "kayobe"
+activate_kayobe_env
+
+set +x
+export KAYOBE_VAULT_PASSWORD=$(cat ~/vault-pw)
+set -x
+
+kayobe control host bootstrap
+kayobe seed host configure
+kayobe overcloud host configure
+
+kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/cephadm-deploy.yml
```

Comment:
You may find that Ceph isn't ready when this command completes, and the next playbook fails.